src/commands/app/db/collection/findOne.js

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { DBBaseCommand } from '../../../../DBBaseCommand.js'
+import { Args, Flags } from '@oclif/core'
+import chalk from 'chalk'
+
+export class FindOne extends DBBaseCommand {
+  async run () {
+    const { collection, filter } = this.args
+    const { projection } = this.flags
+
+    try {
+      // Parse filter JSON
+      let filterObj
+      try {
+        filterObj = JSON.parse(filter)
+      } catch (error) {
+        this.error(`Invalid filter JSON: ${error.message}`)
+      }
+
+      // Parse projection JSON if provided
+      let projectionObj
+      if (projection) {
+        try {
+          projectionObj = JSON.parse(projection)
+        } catch (error) {
+          this.error(`Invalid projection JSON: ${error.message}`)
+        }
+      }
+
+      this.log(chalk.blue(`Finding document in collection '${collection}'...`))
+
+      if (projection) {
+        this.log(chalk.dim(`   Using projection: ${projection}`))
+      }
+
+      const client = await this.db.connect()
+      const coll = client.collection(collection)
+
+      // Build options
+      const options = {}
+      if (projectionObj) {
+        options.projection = projectionObj
+      }
+
+      // Find the document
+      const result = await coll.findOne(filterObj, options)
+
+      this.debugLogger?.info?.('Document found:', result)
+
+      const response = {
+        collection,
+        filter: filterObj,
+        projection: projectionObj,
+        document: result,
+        namespace: this.rtNamespace,
+        timestamp: new Date().toISOString()
+      }
+
+      if (result) {
+        this.log(chalk.green(`Document found in collection '${collection}'`))
+        this.log(chalk.dim(`   Namespace: ${this.rtNamespace}`))
+
+        if (projection) {
+          this.log(chalk.dim('   Projection applied: Yes'))
+        }
+
+        if (!this.flags.json) {
+          this.log(chalk.dim('   Document:'))
+          this.log(chalk.dim(`     ${JSON.stringify(result, null, 2)}`))
```

Comment:
Indent to the whole document to make it prettier

```suggestion
          this.log(chalk.dim(`${JSON.stringify(result, null, 2).replace(/^/gm, '     ')}`))
```